crates/viewer/re_view_spatial/src/transform_cache.rs

```diff
@@ -392,127 +476,173 @@ impl TransformCacheStoreSubscriber {
         }
     }
 
-    fn add_chunk(&mut self, event: &re_chunk_store::ChunkStoreEvent, aspects: TransformAspect) {
+    fn add_non_static_chunk(
```

Comment:
`add_chunk` or `add_temporal_chunk` is what we usual call this

Reply:
much better, thanks